front/src/modules/trainschedule/components/Timetable/Timetable.tsx

```diff
@@ -133,6 +133,13 @@ export default function Timetable({
     }
   };
 
+  const handleConflictClick = (conflict: Conflict) => {
+    if (conflict.train_ids.length > 0) {
+      const firstTrainId = conflict.train_ids[0];
+      dispatch(updateSelectedTrainId(firstTrainId));
+    }
```

Comment:
```suggestion
  const hasTrainIds = conflict.train_ids.length > 0;
  return hasTrainIds ? dispatch(updateSelectedTrainId(conflict.train_ids[0])) : null;
```

Reply:
In handleConflictClick, we don't need to return anything but we do do some things. So no need to move everything in a return statement :)
I think it's perfect like this